src/components/Profile/index.jsx

```diff
@@ -14,6 +14,7 @@ import {
   SchedulePanel,
   VictoryPromise,
 } from './components';
+import { Markup } from 'interweave';
```

Comment:
```suggestion
import { Markup } from 'interweave';
```
what is the use of this import?

Reply:
We left out some code from testing, it is removed now